kubernetes-typescript/package.json

```diff
@@ -5,6 +5,7 @@
     },
     "dependencies": {
         "@pulumi/pulumi": "^1.0.0",
-        "@pulumi/kubernetes": "^1.0.0"
+        "@pulumi/kubernetes": "^1.0.0",
+        "@pulumi/kubernetesx": "^0.1.1"
```

Comment:
I believe both of these can be "latest" - i'm using latest on both in the demo stack for kubecon and have no issues

Reply:
We decided to use `^1.0.0` in templates for things that have hit 1.0 so that users pick up 1.x.y but not 2.x.y automatically.

For pre-1.0 it matters less - but we should be consistent with e.g. https://github.com/pulumi/templates/blob/master/aws-typescript/package.json which @lblackstone's change here is.